### ISSUE TITLE
How to remove uploaded wheel keeping tar.gz present

### ISSUE THREAD

**Issue by vlcinsky** [devpi/devpi#446]
I have uploded a package and later once more, this time as wheel:

```bash
$ devpi list ttr-rcu-mqtt -v
https://pypi.dev.com/javl/dev/+f/85d/f53cd066bd519/ttr-rcu-mqtt-0.0.0.tar.gz
raspberrypi linux2  py27       2.7.9 tests passed
dpo-proto-ow-03.lan linux2  py27       2.7.9 tests passed
https://pypi.dev.com/javl/dev/+f/8db/d598ef0c0a2e0/ttr_rcu_mqtt-0.0.0-py2-none-any.whl
```

Is there any way to remove the wheel package and keep the tar.gz present?

One reason is, that using `$ devpi test ttr-rcu-mqtt==0.0.0` fails with "Only universal wheels are supported".

**Comment by fschulze** [comment]
Not a fix yet, but see #448

**Comment by fschulze** [comment]
With devpi-server 4.6.0 a single release can be deleted using a DELETE request on it's URL. For now one has to use curl or similar tool, which is a bit cumbersome because of authentication etc. I plan to add support for it in devpi-client.
